modules/indexing/src/test/java/org/apache/ignite/sqltests/SqlDataTypesCoverageTests.java

```diff
@@ -366,7 +365,9 @@ private void checkCRUD(IgniteEx ignite, String tblName, SqlDataType dataType, Ob
             if (writeSyncMode == CacheWriteSynchronizationMode.FULL_ASYNC &&
                 !waitForCondition(() -> ignite.context().query().querySqlFields(
                     new SqlFieldsQuery("SELECT val FROM " + tblName), false).getAll().stream()
-                        .allMatch(r -> r.get(0).equals(expRevertedVal)),
+                        .allMatch(r -> r.get(0) instanceof byte[]
```

Comment:
Might be an external comparing function or a Comparator would look more readable. Up to you.